core/system/inc/mpu/vmpu.h

```diff
@@ -101,8 +101,17 @@ static UVISOR_FORCEINLINE int vmpu_sram_addr(uint32_t addr)
 #define VMPU_SRAM_BITBAND_START   0x22000000
 #define VMPU_SRAM_BITBAND_END     0x23FFFFFF
 #define VMPU_PERIPH_START         0x40000000
+#define VMPU_PERIPH_END           0x40100000
+#define VMPU_PERIPH_MASK          0xFFF00000
 #define VMPU_PERIPH_BITBAND_START 0x42000000
 #define VMPU_PERIPH_BITBAND_END   0x43FFFFFF
+#define VMPU_PERIPH_BITBAND_MASK  0xFE000000
+#define VMPU_PERIPH_FULL_MASK     0xFC000000
+
+/* ROM Table region boundaries */
+#define VMPU_ROMTABLE_START 0xE00FF000UL
+#define VMPU_ROMTABLE_END   0xE00FFFFFUL
+#define VMPU_ROMTABLE_MASK  0xFFFFF000UL
 
```

Comment:
@AlessandroA remove unused  VMPU_PERIPH_END and VMPU_PERIPH_BITBAND_END and VMPU_ROMTABLE_END symbols ? Create the wrong impression that the end can be non-aligned.